nuxt.config.ts

```diff
@@ -66,3 +72,5 @@ export default {
     }
   }
 }
+
+module.exports = nuxtConfig
```

Comment:
```suggestion
export default nuxtConfig
```

Reply:
Thank you for pointing out!
This has been fixed.